cropduster/models.py

```diff
@@ -493,7 +493,12 @@ def _save_standalone_thumb(self, size, image=None, thumb=None):
         with default_storage.open(thumb_path, mode='rb') as f:
             md5.update(f.read())
         thumb.name = md5.hexdigest()[0:9]
-        os.rename(thumb_path, self.get_image_path(thumb.name))
+        old_image_path = thumb_path
+        new_path = self.get_image_path(thumb.name)
```

Comment:
Rather than repeat this open here, perhaps the above one can be changed to

```python
with default_storage.open(thumb_path, mode='rb') as f:
    image_contents = f.read()
md5.update(image_contents)
with default_storage.open(new_path, 'wb') as f:
    f.write(image_contents)
default_storage.delete...
```

Reply:
A-ha, yeah 👍